include/sdsl/int_vector.hpp

```diff
@@ -648,6 +648,26 @@ class int_vector {
 	//! Load the int_vector for a stream.
 	void load(std::istream& in);
 
+	// //!\brief Serialise (save) via cereal if archive is not binary
```

Comment:
Can you make a small comment and explain why different code is necessary? maybe a reference to the `std::vector` serialization of cereal?

Reply:
I'll add this once the CI judged the code.

Reply:
Ok. And I noticed `//` is a duplicate